app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if "mv" in command:
+        cmd, origin_file, path = command.split()
+        if "/" in path:
```

Comment:
use os.path.split

Reply:
but command os.path.split don`t work with simple strings and I used this in the lower string with the path and folder name